ICA/auto_Granduer_ICA.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import json
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date,datetime
+
+# This funciton uses the ica CLI tool to return info from ICA
+def icav_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('\t', ' ') for x in tmp]
+    return tmp
+
+# While loop is infinate will keep script running as long as it dosent error out.
+go=1
+while go == 1:
+    print('Looking for anaylses that need to start!')
```

Comment:
typo: analysis != anaylses

Reply:
Spelled wrong but still needs to be plural!